### PR TITLE
KEYCLOAK-19348 SAML/ OIDC IdP AutoUpdate

### PR DESCRIPTION
**Pull request by cgeorgilakis**
https://github.com/keycloak/keycloak/pull/8451

### CODE REVIEW

File: server_admin/topics/identity-broker/autoupdated.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+[[_mappers]]
+=== Automatic Updated SAML/ OIDC Identity Providers
+
+SAML and OIDC Identity Providers can be configured to automatically updated from its metadata URL (during Identity Provider creation and update).
```
Comment (andymunro) [inline server_admin/topics/identity-broker/autoupdated.adoc]: add "be" before "updated"
change "its" to "their"

File: server_admin/topics/identity-broker/autoupdated.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Autoupdate field was added (default value false). When autoupdate is true, realm admin must fulfill Refresh period field( default 1 day).
```
Comment (andymunro) [inline server_admin/topics/identity-broker/autoupdated.adoc]: add "the" before 3 occurrences of "realm admin"

File: server_admin/topics/identity-broker/autoupdated.adoc [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+For SAML Identity Provider, realm admin must fulfill Metadata URL.
+For OIDC Identity Provider, realm admin must fulfill Issuer. Metadata URL automatically become equal to issuer + '/.well-known/openid-configuration'.
```
Comment (andymunro) [inline server_admin/topics/identity-broker/autoupdated.adoc]: change "become" to "becomes"

File: server_admin/topics/identity-broker/autoupdated.adoc [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+image:{project_images}/auto-updated.png[]
+
+In this case, SAML/OIDC Identity Provider is automatically updated every Refresh period. Last refresh time is automatically filled and shows last Identity Provider autoupdate.
```
Comment (andymunro) [inline server_admin/topics/identity-broker/autoupdated.adoc]: Add "the" before two occurrences of "last"

**Review by andymunro — CHANGES_REQUESTED** [review]
A few comments. Thanks.

**Comment by cgeorgilakis** [comment]
I have done proposed changes and rebase to current main branch.

**Review by andymunro — APPROVED** [review]
@cgeorgilakis Thanks for the quick response.

**Comment by ahus1** [comment]
The documentation has been moved to [the main Keycloak repository](https://github.com/keycloak/keycloak/tree/main/docs/documentation).

Please include these documentation changes into the main pull request.

Thanks!

**Pull request closed without merging**